api/storage/app/lighthouse-schema.graphql

```diff
@@ -153,7 +153,7 @@ enum CandidateExpiryFilter {
 }
 
 type CandidateSearchPoolResult {
```

Comment:
not entirely sure where this issue comes from, but requests are broken for `request-responders`
observe, as admin the requests page works, but removing the admin role and keeping responder breaks the dashboard page

https://recordit.co/7MbVH4BKAX

the requests are still fetched though, the error path ends at pools, but could be a teams thing

![image](https://user-images.githubusercontent.com/40485260/226738676-adcbdf15-44a1-45ac-b6eb-036570787927.png)

Reply:
oh yeah, big disclaimer, I merged in main for my local environment before testing
to really be careful for this

Reply:
no issues with this on main
guessing it is due to changes to 

![image](https://user-images.githubusercontent.com/40485260/226759741-84f5e62e-4bd7-4ec4-99d0-c1e3f79a738b.png)

Reply:
This is going to require either giving `request-responders` access to `view-any-pool` or attempting to change this over to use PoolAdvertisements instead (in theory any pool in a request should be published).

@tristan-orourke Can you provide input on this?

Reply:
We're aware this is an issue, and it will be fixed in #5847. I'm not worried about this making its way into production in the meantime, because all our admin users currently have all three admin roles.